app/src/main/java/me/eugeniomarletti/renderthread/CanvasUtils.java

```diff
@@ -0,0 +1,26 @@
+package me.eugeniomarletti.renderthread;
+
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.RectF;
+import android.os.Build;
+import android.support.annotation.NonNull;
+
+public final class CanvasUtils {
+
+    private CanvasUtils() {
+    }
+
+    private static final RectF TMP_RECT = Build.VERSION.SDK_INT < 21 ? new RectF() : null;
+
+    public static void drawRoundRect(
```

Comment:
Can you reformat it? It looks a bit strange with the open ( and the parameters all one line